sass/base/_reset.scss

```diff
@@ -0,0 +1,25 @@
+* {
+  box-sizing: border-box;
+}
```

Comment:
See [this article](https://css-tricks.com/inheriting-box-sizing-probably-slightly-better-best-practice/) for a slightly better `box-sizing` pattern.